Updates for modified property

ehanson8 opened the pull request:
#### What does this PR do?
Updates to account for new modified property

#### Includes new or updated dependencies?
NO

ehanson8 committed:
updates for modified property

gravesm commented on asaps/models.py:
```diff
@@ -156,15 +154,16 @@ def filter_note_type(client, csv_data, rec_obj, note_type, operation,
             if note['type'] == note_type:
                 for subnote in note['subnotes']:
                     if operation == 'replace_str':
                         fieldval = subnote['content']
```
Since we don't need to worry anymore about whether or not a specific field has changed, I think we can just do the replace right here and remove a bunch of code:
```python
subnote['content'] = subnote['content'].replace(old_string, new_string)
```

ehanson8 committed:
simplify replace code

Simplified but I'm preserving the original value for the logs as a part of the replacestr operation, this already proved useful in showing DDC some issues with their originally suggested strings to be replaced
